### ISSUE TITLE
support replying to soapbox threads with multiple users mentioned

### ISSUE THREAD

**Issue by 900809** [syuilo/Misskey#12954]
### Summary

in soapbox the user handle is not part of the message:

![image](https://github.com/misskey-dev/misskey/assets/104532529/2fd5ced2-9ecd-48ec-aa88-4cef66fbb551)

so when a misskey user replies to a soapbox user who is mentioning multiple people, all the tags are dropped except the person who the reply is addressed to:

![image](https://github.com/misskey-dev/misskey/assets/104532529/45989cd8-0e00-415f-8c00-317bba3a6f80)
![image](https://github.com/misskey-dev/misskey/assets/104532529/72998266-7311-42df-8b65-f914b44380ac)
(missing 1 person's handle)

it would be good if misskey could see everyone mentioned in the thread and add their handles to the reply automatically like:

![image](https://github.com/misskey-dev/misskey/assets/104532529/1003ae8c-8b12-4bcf-8a00-cf453eae413a)

### Purpose

at the moment threads where both misskey and soapbox users are participating in are broken and hard to follow. soapbox users mentioned in the thread who are not directly being replied to by a misskey user are dropped from the thread and don't get notifications of replies. this is confusing and adding support for the soapbox reply system would fix this

### Do you want to implement this feature yourself?

- [ ] Yes, I will implement this by myself and send a pull request

**Comment by temtemy** [comment]
IMO I think it should be Soapbox's side fixing this, not Misskey. Because how would you differentiate between a Soapbox reply and a reply where the poster *intentionally* removed the mentions?

**Comment by 900809** [comment]
> IMO I think it should be Soapbox's side fixing this, not Misskey. Because how would you differentiate between a Soapbox reply and a reply where the poster _intentionally_ removed the mentions?

i don't disagree to be honest. this has been complained about for a while and ive always said it's probably a soapbox issue but it ws suggested that not being able to see replies that are not in plaintext was an issue on misskey's side so i decided to open an issue in case devs were not aware. feel free to close if theres no interest in implementing this
